.github/workflows/SwiftFormat.yml

```
@@ -0,0 +1,76 @@
+name: SwiftFormat
+
+on:
+  workflow_dispatch:
+
+jobs:
+  windows:
+    runs-on: windows-latest
+
+    strategy:
+      matrix:
+        include:
+          - tag: 0.51.10
+            branch: master
+
+    steps:
+      # Build
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          ref: refs/tags/${{ matrix.tag }}
+          repository: nicklockwood/SwiftFormat
+
+      - uses: compnerd/gha-setup-swift@main
+        with:
+          branch: swift-5.8-release
+          tag: 5.8-RELEASE
+
+      - name: test
+        run: |
+          swift test -Xcc -IC:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\include -Xlinker -LC:\Library\Developer\Toolchains\unknown-Asserts-development.xctoolchain\usr\lib
```

```suggestion
          swift test
```

Do we really need any of the additional include and library search paths?

---

Done